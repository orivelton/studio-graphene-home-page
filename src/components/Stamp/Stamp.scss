@import '../../assets/scss/helper.scss';

.stamp {
  width: 150px;
  height: 150px;
  border-radius: 50%;
  display: flex;
  justify-items: center;
  position: relative;
  justify-content: center;
  align-items: center;
  color: $night-rider;
  position: absolute;
  top: 11%;
  left: 15%;
  &::before {
    font-size: 150px;
    position: absolute;
  }

  &__text {
    font-weight: bold;
    font-family: 'DIN Condensed';
    font-size: 52px;transform: rotate(-34deg);
    margin-top: 11px;
    display: flex;
    width: 100%;
    justify-content: center;
  }
}
