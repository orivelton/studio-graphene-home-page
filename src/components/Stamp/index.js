import React from 'react';
import './Stamp.scss'

const Stamp = ({textTop, text, textBottom}) => (
  <div className="stamp uppercase icon iconstamp">
    <p className="stamp__text">{text}</p>
  </div>
);

export default Stamp;
