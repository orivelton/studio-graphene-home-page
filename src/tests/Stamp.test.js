import React from 'react';
import renderer from 'react-test-renderer';
import Stamp from '../components/Stamp';

test('renders correctly Stamp', () => {
  const tree = renderer.create(<Stamp />).toJSON();
  expect(tree).toMatchSnapshot();
});
