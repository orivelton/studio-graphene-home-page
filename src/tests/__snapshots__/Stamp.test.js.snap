// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Stamp 1`] = `
<div
  className="stamp uppercase icon iconstamp"
>
  <p
    className="stamp__text"
  />
</div>
`;
